src/services/iiif-parser.js

```diff
@@ -580,9 +581,17 @@ export function getStructureRanges(manifest) {
       if (isCanvas && canvasInfo.range != undefined) {
         duration = canvasInfo.range.end - canvasInfo.range.start;
       }
+      if (isCanvas) {
+        let parsedManifest = parseManifest(manifest);
+        let summaryProperty = parsedManifest.getSequences()[0].getCanvasById(getCanvasId(canvases[0]))?.getProperty('summary');
```

Comment:
Instead of importing `getSequences` from manifest.js and parsing the manifest here, I think we can just use the `parseSequences` function from the utility-helpers file which is already imported.
```suggestion
        let summaryProperty = parseSequences(manifest)[0].getCanvasById(getCanvasId(canvases[0]))?.getProperty('summary');
```